.gitignore

```diff
@@ -9,3 +9,7 @@ ssl
 Cargo.toml
 target
 Cargo.lock
+target
+Cargo.toml
+androidapp/Crisislab/app/src/androidTest/
+androidapp/Crisislab/app/src/test/
```

Comment:
Why do we need these here if they're in the dedicated .gitignore?

Reply:
Fixed, will push soon